gen-rpm.in

```diff
@@ -0,0 +1,174 @@
+#!/bin/sh
+
+# Export filepaths
+export BUILDDIR=@CMAKE_CURRENT_SOURCE_DIR@/build
+export BUILDROOT=~/rpmbuild/
+export RPMSRC=~/rpmbuild/SOURCES
+export RPMSPEC=~/rpmbuild/SPECS
+export RPMBUILD=~/rpmbuild/BUILD
+
+# Check for Docker switch
+if [ "$1" == "-d" ]; then
+	export DOCKERSTATUS=TRUE
```

Comment:
I'm guessing the variable isn't being set how we think.

Or do we need to wrap TRUE in quotes? `"TRUE"`

Just guessing. Not a bash expert.

Reply:
Running your last commit now. That's the last one I can do tonight.

Appreciate your patience on this! We're close

Reply:
It shouldn't.

```
 h  /tmp  cat test.sh
if [ "$1" == "-d" ]; then
	export DOCKERSTATUS=TRUE
else
	export DOCKERSTATUS=FALSE
fi

echo $DOCKERSTATUS
 h  /tmp  bash test.sh
FALSE
 h  /tmp  bash test.sh -d
TRUE
 h  /tmp  
```